managed-services/transactional-email/reference-content/smtp-configuration.mdx

```diff
@@ -0,0 +1,65 @@
+
+On this page we go through different elements you might find in a standard SMTP configuration, and how to fill out each entry to ensure an optimal configuration of your Transactional Email service.
+
+## Standard Settings
+
+1 - **SMTP Server Address (Host)** - The domain name or IP address of the SMTP server. If you are using Scaleway Transactional Email, the domain to use is `smtp.tem.scw.cloud`.
+
+2 - **SMTP Port** - The port number used to connect to the SMTP server. At Scaleway, the default ports are `25`, `587`, and `2587`.
+
+  Depending on your use case, we recommend using the following ports:
+
+   - **Port 587**: Used to securely submit emails over TLS.
+   - **Port 465**: Used for SMTP connections over SSL/TLS. At Scaleway you can also use port `2465`.
+   - **Port 25**: This port was traditionally used for relaying and submitting emails, but it is not recommended when sending emails directly from an application, due to widespread blocking to prevent spam.
+
+3 - **Username and Password** - Sometimes you need to authenticate to the SMTP server to connect. You might be asked to provide a valid username and password.
+
+Your Scaleway SMTP username is the Project ID of the Project in which the TEM domain was created. You can find it under **Managed Services** > **Transactional Email** > **Domain Overview** in the Scaleway console.
+
+Your password is the secret key of the API key of the project used to manage your TEM domain. Follow this procedure [to generate API keys for API and SMTP sending with IAM](https://www.scaleway.com/en/docs/managed-services/transactional-email/how-to/generate-api-keys-for-tem-with-iam/).
+
+4 - **Encryption Method** - If you want to Encrypt the connection between your application and the SMTP server, there are usually two methods available:
+
+   - **SSL/TLS**: Also known as SMTPS, it allows you to directly define a secure connection on a secure port. Directly creates a secure connection on a port such as `465` and `2465`.
+   - **STARTTLS**: This type will upgrade any insecure connections to secure connections on a non-secure port, such as 587.
+
+5 - **Sender Email Address** - The email address that will appear as the sender. Whenever someone receives an email from your application this is the email they see.
+
+6 - **Authentication Method** - This is the method you can use to ensure that only authorized users can send emails from the SMTP server, which helps avoid spam.
+
+SMTP supports different methods:
+
+    - **PLAIN**: Username and password are transmitted in plain text and secured via SSL/TLS.
+    - **LOGIN**: Username and password are encoded in base64 and then transmitted through SSL/TLS.
+    - **CRAM-MD5**: The most secure method where a challenge-response mechanism is used, to avoid sending the password via a network.
```

Comment:
We are not supporting yet CRAM-MD5 login mechanism.

Note that the LOGIN method is considered as Obsolete by the iana https://www.iana.org/assignments/sasl-mechanisms/sasl-mechanisms.xhtml